src/RelataBH/RelataBH/Service/Profile/RelatoHistoric/IRelatoHistoricServiceImpl.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RelataBH.Database;
+
+namespace RelataBH.Service.Profile.RelatoHistoric
+{
+    public class IRelatoHistoricServiceImpl(RelatoContext relatoContext) : IRelatoHistoricService
+    {
+        public async Task<IEnumerable<Model.Relato.Relato>> GetRelatosByUser(int userId)
+        {
+            var relatosUser = await relatoContext.Relatos
+               .Where(x => x.idUser == userId)
```

Comment:
no historico seria massa se retornasse a lista de relato do mais recente pro mais antigo (pra isso da pra usar o OrderByDescending)

```suggestion
               .Where(x => x.idUser == userId)
               .OrderByDescending(item => item.createdAt)
```